h2o-core/src/main/java/water/api/ModelsHandler.java

```diff
@@ -256,4 +256,21 @@ public ModelExportV3 exportMojo(int version, ModelExportV3 mexport) {
     return mexport;
   }
 
+  public ModelExportV3 exportModelDetails(int version, ModelExportV3 mexport){
+    Model model = getFromDKV("model_id", mexport.model_id.key());
+    try {
+      URI targetUri = FileUtils.getURI(mexport.dir); // Really file, not dir
+      Persist p = H2O.getPM().getPersistForURI(targetUri);
+      //Output model details to JSON
+      OutputStream os = p.create(targetUri.toString(),mexport.force);
+      os.write(model.writeJSON(new AutoBuffer()).buf());
```

Comment:
Sure, it would be something like the following: 
`{"_key":{"name":"GBM_model_R_1486170348617_1","type":"Key"},"_checksum":3497837044092317208,"_parms":{"_train":{"name":"iris","type":"Key"},"_valid":null,"_nfolds":0,"_keep_cross_validation_predictions":false,"_keep_cross_validation_fold_assignment":false,"_parallelize_cross_validation":true,"_auto_rebalance":true,"_seed":-4150278781309862746,"_fold_assignment":"AUTO","_categorical_encoding":"AUTO","_distribution":"multinomial","_tweedie_power":1.5,"_quantile_alpha":0.5,"_huber_alpha":0.9,"_ignored_columns":[""],"_ignore_const_cols":true,"_weights_column":null,"_offset_column":null,"_fold_column":null,"_is_cv_model":false,"_score_each_iteration":false,"_max_runtime_secs":0.0,"_stopping_rounds":0,"_stopping_metric":"AUTO","_stopping_tolerance":0.001,"_response_column":"Species","_balance_classes":false,"_max_after_balance_size":5.0,"_class_sampling_factors":null,"_max_confusion_matrix_size":20,"_checkpoint":null,"_pretrained_autoencoder":null,"_ntrees":50,"_max_depth":5,"_min_rows":10.0,"_nbins":20,"_nbins_cats":1024,"_min_split_improvement":1.0E-5,"_histogram_type":"AUTO","_r2_stopping":1.7976931348623157E308,"_nbins_top_level":1024,"_build_tree_one_node":false,"_score_tree_interval":0,"_initial_score_interval":4000,"_score_interval":4000,"_sample_rate":1.0,"_sample_rate_per_class":null,"_col_sample_rate_change_per_level":1.0,"_col_sample_rate_per_tree":1.0,"_learn_rate":0.1,"_learn_rate_annealing":1.0,"_col_sample_rate":1.0,"_max_abs_leafnode_pred":1.7976931348623157E308,"_pred_noise_bandwidth":0.0},"_output":{"_names":["Sepal.Length","Sepal.Width","Petal.Length","Petal.Width","Species"],"_origNames":null,"_domains":[null,null,null,null,["setosa","versicolor","virginica"]],"_origDomains":null,"_cross_validation_models":null,"_cross_validation_predictions":null,"_cross_validation_holdout_predictions_frame_id":null,"_cross_validation_fold_assignment_frame_id":null," ...`